elm.json

```diff
@@ -8,11 +8,11 @@
         "direct": {
```

Comment:
Don't forget to update to 0.19.1 in the elm.json too!

Reply:
done

Reply:
You still need to push this change